.travis.yml

```diff
@@ -3,6 +3,9 @@ language: python
 python:
   - "2.7"
 
+notifications:
+  email: false
```

Comment:
@Kennric Is there a way for you to turn it off as an admin? I can't find an option.